<launch>

  <node name="rviz"
    pkg="rviz"
    type="rviz"
    output="log"
    args="-d $(find mav_coverage_planning_ros)/cfg/rviz_2d_shortest_path.rviz" />


  <node name="shortest_path_planner_2d"
    pkg="mav_coverage_planning_ros"
    type="shortest_path_2d_node"
    output="screen"
    clear_params="true" >

    <!-- Basic and control parameter -->
    <rosparam command="load" file="$(find mav_coverage_planning_ros)/cfg/base_planner_2d.yaml" />
    <!-- Polygon, altitude, and world frame ID -->
    <rosparam command="load" file="$(find mav_coverage_planning_ros)/cfg/example_polygon_forest.yaml" />
    <param name="robot_size" value="1.0" />
    <param name="min_view_overlap" value="0.2" />
  </node>

</launch>
